script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -303,17 +303,18 @@ His body isn't falling apart and his %{i}Principle%{/i} is fairly small-scale. H
 [sha:a9b0504bc7cc4ba28627cb01d48312fbb7ee6c1a]{
 -- Page 166, Offset 8990.
 -- ……と。アルクェイドのヤツ、とんでもなく露骨にため息をこぼしやがった。
-...As soon as I say that, Arcueid lets out an exaggerated sigh.
+As soon as I say that... Arcueid lets out an exaggerated sigh.
 }
 [sha:ff4e00befb382741ba1f9fa9266f79646ee89965]{
 -- Page 167, Offset 8991.
 -- 「そりゃあね。誰かさんに殺されてしまったわたしは、あんな二流の死徒と“戦い”をするレベルまで落ちてるわよ。
-"Well, you see. I was killed by a certain somebody, which made me drop to such a level where I have to 'fight' even a second-rate Dead Apostle like him.
+"Well, you see... Thanks to a certain someone killing me, I've dropped to a level where even when going up against a second-rate Dead Apostle like that, it would actually qualify as a 'fight'.
```

Comment:
where even going up against a second-rate Dead Apostle like that would qualify as a 'fight'.